components/tekton/pipelineruns/run-sync-test-suite/run-test-suite.yaml

```diff
@@ -0,0 +1,23 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: run-vse-sync-testsuite-run
+  labels:
+    tekton.dev/pipeline: run-vse-sync-testsuite
+spec:
+  pipelineRef:
+    name: run-vse-sync-testsuite
+  serviceAccountName: pipeline-credentials-sa
+  timeout: 1h0m0s
```

Comment:
Did you pass the default timeout? just wondering if there is a need for specifying timeout instead of using the default

Reply:
Good point I'll remove it :)